recovery_behaviors:
    - name: 'clear_costmap'
      type: 'clear_costmap_recovery/ClearCostmapRecovery'
